core/src/main/java/io/confluent/kafka/schemaregistry/rest/client/LocalSchemaRegistryClient.java

```diff
@@ -263,7 +263,8 @@ public synchronized List<ParsedSchema> getSchemas(
       schemas = schemaRegistry.getVersionsWithSubjectPrefix(
           subjectPrefix,
           lookupDeletedSchema ? LookupFilter.INCLUDE_DELETED : LookupFilter.DEFAULT,
-          latestOnly);
+          latestOnly,
+          null);
```

Comment:
isn't LocalSchemaRegistry client not used anywhere now? wondering why we skipped ruleType support here.